keda/templates/01-serviceaccount.yaml

```diff
@@ -2,6 +2,8 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
+  annotations:
+    {{- toYaml .Values.additionalAnnotations | nindent 4 }}
```

Comment:
move this to the already existing `annotations` block

Reply:
Done